### PR TITLE
Typo fixes in in ./build and ./lib

### PR DESCRIPTION
**Pull request by luzpaz**
+ User facing typos
+ trivial source comment typos
+ superflous double whitespace issues
Found using `codespell -q 3 --skip="./build/wix/Localization,./res/translations" -I ../mixxx-whitelist.txt` where the contents of the whitelist were:
```
iff
freee
substract
```

### CODE REVIEW

File: lib/soundtouch-2.0.0/RateTransposer.h [hunk 1]
```diff
@@ -138,7 +138,7 @@ class RateTransposer : public FIFOProcessor
 
     /// Use this function instead of "new" operator to create a new instance of this class. 
     /// This function automatically chooses a correct implementation, depending on if 
-    /// integer ot floating point arithmetics are to be used.
+    /// integer or floating point arithmetics are to be used.
```
Comment (luzpaz) [inline lib/soundtouch-2.0.0/RateTransposer.h]: Is this correct ?

**Comment by Be-ing** [comment]
Fixing typos in the build directory is fine, but we should not modify files from external libraries without a pressing reason.

**Comment by luzpaz** [comment]
@Be-ing indeed which libraries are external?

**Comment by Be-ing** [comment]
Everything in the `lib` folder is an external library.

**Comment by luzpaz** [comment]
@Be-ing A-OK my bad. I'll close this PR then and then resubmit.

**Comment by luzpaz** [comment]
#1417

**Comment by daschuer** [comment]
Thank you for helping here. 
Unfortunately these kind of mass changes in 55 files leads probably to a lot of conflicts in our pending PRs. So we have to carefully consider if it is worth to commit this PR now, producing extra work in the open PRs.  I am curious to see the final PR for Mixxx code only.
Maybe it is a good time to do it just before our beta deadline.

**Comment by luzpaz** [comment]
I'm not attached. Feel free to do as you wish.

**Pull request closed without merging**